### PR TITLE
Add mono_lesson? method to topic

### PR DESCRIPTION
**Pull request by fedescarpa**

### CODE REVIEW

File: app/models/topic.rb [hunk 1]
```diff
@@ -35,6 +35,10 @@ def reindex_usages!
     Chapter.where(topic: self).map(&:book).each(&:reindex_usages!)
   end
 
+  def mono_lesson?
+    lessons.size == 1
```
Comment (luchotc) [inline app/models/topic.rb:39]: Prefer count as it's an AR relation. I'll change it :smile:

**Comment by flbulgarelli** [comment]
Looks good!

My only comment is that I expected `monolesson` since its a single word, not two - `mono-` is just a prefix as in `monorail` or `monolithic`

**Comment by luchotc** [comment]
@flbulgarelli I could use a re-review :smile:

File: app/models/chapter.rb [hunk 2]
```diff
@@ -13,6 +13,9 @@ class Chapter < ApplicationRecord
 
   has_many :exercises, through: :topic
 
+  delegate :monolesson?, :monolesson, :first_lesson, to: :topic
+
+  delegate :next_exercise, :stats_for, to: :monolesson, allow_nil: true
```
Comment (flbulgarelli) [inline app/models/chapter.rb:18]: Nice

**Review by flbulgarelli — APPROVED** [review]

**Pull request merged**